R/utils.R

```diff
@@ -200,3 +200,47 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print information about a `forecast` object.
+#' @description This function prints information about a forecast object,
+#' including "Forecast type", "Score columns",
+#' "Forecast unit".
+#' @export
+#' @keywords print-forecast
+#' @examples
+#' print(example_quantile %>% as_forecast)
```

Comment:
This line produces an error since magrittr is not automatically part of the namespace. 

I would suggest to simply replace this by 
```suggestion
#' as_forecast(example_quantile)
```

Reply:
Since this is a print function, we need to write as `print(as_forecast(example_quantile))` ?

Reply:
Done in the commit [d3a75cd](https://github.com/epiforecasts/scoringutils/pull/592/commits/d3a75cda48646d206673faba8ddc90e8510cf536)